src/Sylius/Bundle/ThemeBundle/Command/AssetsInstallCommand.php

```diff
@@ -33,7 +33,7 @@ class AssetsInstallCommand extends ContainerAwareCommand
     protected function configure()
     {
         $this
-            ->setName('sylius:assets:install')
+            ->setName('sylius:theme:assets:install')
```

Comment:
Why? I do prefer not overriding Symfony commands of course, but will it require the developer to execute 2 commands instead of 1?